src/main/java/org/usfirst/frc4904/robot/commands/ElevatorSetSpeed.java

```diff
@@ -0,0 +1,38 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Elevator;
+import org.usfirst.frc4904.standard.commands.RunIf;
+import org.usfirst.frc4904.standard.commands.RunIfElse;
+import org.usfirst.frc4904.standard.commands.motor.MotorControl;
+import org.usfirst.frc4904.standard.custom.controllers.CustomJoystick;
+
+/**
+ * Control elevator manually
+ */
+public class ElevatorSetSpeed extends RunIfElse {
+	public ElevatorSetSpeed(CustomJoystick joystick) {
+		super(
+			new RunIf(new MotorControl(RobotMap.Component.elevator, joystick, CustomJoystick.Y_AXIS,
+				Elevator.UP_SPEED),
+				() -> {
+					return RobotMap.Component.elevator.getPosition() < Elevator.MAX_POSITION
+						|| RobotMap.Component.elevator.isOverridden();
+				}),
+			new RunIf(new MotorControl(RobotMap.Component.elevator, joystick, CustomJoystick.Y_AXIS,
```

Comment:
why use the joystick if we don't scale the speed? I would say either scale speed with joystick or set to an up button and a down button